### ISSUE TITLE
World creation is confusing. New users get stuck.

### ISSUE THREAD

**Issue by WizardOfOoo** [CleverRaven/Cataclysm-DDA#18394]
New users are often confused by the need to hit > on the world creation screen.
I myself experienced this problem when I was new.
I see this question a lot on Reddit and other places.

![image](https://cloud.githubusercontent.com/assets/17521661/18611818/b057424e-7d0c-11e6-93b5-7435a4f0deaf.png)

Could the key be changed to a more uniform one?  Or maybe a prominent help text could be added?

**Comment by WizardOfOoo** [comment]
Yes, they can hit ? for help and find the correct key that way.  But I don't think users should have to use the help menu before they even start playing.

**Comment by Karlieb** [comment]
I remember this screen! I was so happy when I finally gained a level and was able to progress to World Gen Options.

**Comment by WizardOfOoo** [comment]
How about tab?

And a big "press tab to go to next menu"

**Comment by DangerNoodle** [comment]
We already use tab for the mods screen pages though. And unlike with < and >, needing to hit tab was what tripped me up, personally.

Since "backtab" is not as intuitive as < is, this system would also make the world generation menu one-way for most new players.

**Comment by illi-kun** [comment]
Well, tab is using for navigation in mod tabs (as DangerNoodle correctly said) and keybinding is available in help menu (and hint about help menu is already on the screen). What actually do you propose?

**Comment by DangerNoodle** [comment]
We could add additional messages just summarizing the use of <, >, and additionally tab for the mods screen. In the same way that the mods screen mentions hitting s to save the mod selection.

**Comment by remyroy** [comment]
This isn't the only element that would need some serious usability consideration. It would be a good start to fix this one. Here are a few suggestions:

* Change the default key to be `Tab ↹` for the main tabs selection and use `<` `>` for the mod sub-tabs.
* Add mouse support for the menus in SDL version.
* When the right arrow is pressed and the `Mod Load Order` section is selected, go to the *World Gen Options* tab.

**Comment by DangerNoodle** [comment]
Reminder: you expect new users to be able to advance back and forth through the main world creation menu with ↹?

**Comment by remyroy** [comment]
If mouse support and right arrow key is implemented as well, there would be at least 3 ways to change from one main tab to another. I do not expect all new users to be able to advance using the `Tab ↹`. These are just suggestions.

**Comment by WizardOfOoo** [comment]
Don't get hung up on tab guys, It could be ◘ and ñ for all I care.  I just want new players to have a better experience.

**Comment by DangerNoodle** [comment]
That might make it easier, yes. However, if I recall left and right arrow keys in world creation are already reserved for changing settings in the world/city settings section, and for switching back and forth between selecting mods and removing them in the mods section.

Regardless of whether any keybinding changes are made, clarifying them in the world creation screen might be a better first option.

**Comment by WizardOfOoo** [comment]
Agreed.

**Comment by remyroy** [comment]
Usability's motto is **Don't make me think**. If it can work without having to read, it should be a superior usability solution than one that involves reading because you have to think when reading (albeit some small amount of thinking is needed for some people to read, that amount is greater than just mindlessly pushing button for most people).

I'm not a usability expert. I'm just regurgitating [the small amount](https://www.amazon.com/Dont-Make-Think-Revisited-Usability/dp/0321965515) I [learned](https://www.amazon.com/Design-Everyday-Things-Revised-Expanded/dp/0465050654) in [my spare](https://www.amazon.com/Inmates-Are-Running-Asylum-Products/dp/0672326140/ref=sr_1_1?s=books&ie=UTF8&qid=1474231414&sr=1-1&keywords=The+Inmates+Are+Running+the+Asylum) time [reading](https://www.amazon.com/Designing-Web-Usability-Jakob-Nielsen/dp/156205810X).

**Comment by codemime** [comment]
As far as I can say, the essence of the problem is:  we have nested widgets (tabs in this case), but don't have overall "focused widget" mechanism. So we have to use different keys for different widgets (more widgets - more troubles).

So why don't we either:
- Introduce the mechanism (not so convenient anyway)
- Avoid such cases: get rid of the nested tabs and replace them with a flat structure (traverse using arrows like it's done in the main menu):

\<Default\> **\<Blacklist\>** \<Balance\>       \<Load order\>

**Comment by DangerNoodle** [comment]
I am not sure which method would be most usable. For me, my prior experience with roguelikes made < and > natural choices to try once I first realized that the arrow keys served a different purpose. For new players, I am not sure what the most intuitive choice would be. That said, stating up front what the most important keys are does guard against problems caused by unintuitive controls.

Then again it is also the lazy way around this issue.

**Comment by mugling** [comment]
> Usability's motto is Don't make me think.

Needing instructions about how to use a menu strongly suggest the menu is badly designed

**Comment by codemime** [comment]
@remyroy Thanks for the links.

**Comment by Karlieb** [comment]
I remember tab working for all the menus except that one (which was what made this noob confused). I think I ended up going to the wiki? I can't remember. Couldn't it be tab and shift-tab (forward and backward)?

**Comment by fr1gid** [comment]
Considering how the arrow keys are introduced as the method of navigating the main menu, it seems to me that the most natural method would be to continue with them as the primary method of navigation. Then use tab or whatever else to go between other sub-pages. If those subpages aren't being completely redesigned.

**Comment by WizardOfOoo** [comment]
Here's another complaint.

https://www.reddit.com/r/cataclysmdda/comments/5cyc4n/stupid_question/

**Comment by WizardOfOoo** [comment]
I think the world creation menu should be made to look like the settings window.  Have an Enable/Disable option for each mod.  Allow tab to switch categories.  Hitting escape gives the options to  (create world, change settings, or cancel world creation and return to main menu)

**Comment by Leland** [comment]
I'm absolutely leaving this open because I remember my extreme frustration over this as well, but also see #12112.

UI consistency – and even more importantly, conveying those standards – is of the utmost importance.

**Comment by AdriaanRenting** [comment]
This was indeed really confusing when I tried the game for the first time yesterday.

? doesn't help as it seems to say that TAB and BACKTAB should be used for both. (And then also mentions > and < for one of them, but not in a way I understood.)

It's the only part of the interface I had to google so far.
